### PR TITLE
activated email template enhanced

### PR DESCRIPTION
**Pull request by zamandevs**
Companion [PR](https://github.com/weDevsOfficial/wpuf-pro/pull/384)

### CODE REVIEW

File: includes/free/class-login.php [hunk 1]
```diff
@@ -1091,4 +1079,64 @@ public function default_wp_login_override() {
             exit;
         }
     }
+
+    /**
+     * Send activated email for user
+     *
+     * @param $user
+     */
+    private function send_activated_email( $user ) {
```
Comment (tareq1988) [inline includes/free/class-login.php:1088]: The proper function name should be `send_activation_email`

File: includes/free/class-login.php [hunk 2]
```diff
@@ -1091,4 +1079,64 @@ public function default_wp_login_override() {
...
+        $blogname   = wp_specialchars_decode( get_option( 'blogname' ), ENT_QUOTES );
+        $user_email = $user->user_email;
+        $to         = $user_email;
```
Comment (tareq1988) [inline includes/free/class-login.php:1091]: You don't need to declare variables if you don't need to. Both `$user_email` and `$to` variables aren't needed. Use `$user->user_email` directly.

File: includes/free/class-login.php [hunk 3]
```diff
@@ -1091,4 +1079,64 @@ public function default_wp_login_override() {
...
+    /**
+     * Preapring mail body for activated email
+     *
+     * @param $message
+     * @param $user
+     * @param $blogname
+     *
+     * @return string|string[]
+     */
+    private static function prepare_mail_body( $message, $user, $blogname ) {
```
Comment (tareq1988) [inline includes/free/class-login.php:1127]: Do we really need to use a static function here? Unless explicitly necessary, don't make it a static function.

**Review by tareq1988 — CHANGES_REQUESTED** [review]

File: includes/free/class-login.php [hunk 4]
```diff
@@ -1091,4 +1079,62 @@ public function default_wp_login_override() {
             exit;
         }
     }
+
+    /**
+     * Send activated email for user
+     *
+     * @param $user
+     */
+    private function send_email_after_activation( $user ) {
```
Comment (tareq1988) [inline includes/free/class-login.php:1088]: I've asked to name the function as `send_activation_email`, not sure why have you changed it?

Comment (zamandevs) [reply includes/free/class-login.php:1088]: I've thought of activation as when it happens i.e when we sent an email for user account activation. but this email is triggered after activation when the admin approves the user. please advise me if i am doing wrong.

Comment (tareq1988) [reply includes/free/class-login.php:1088]: Still, keep the name as I suggested.

**Pull request merged**